src/components/BaseAppBar.vue

```diff
@@ -0,0 +1,15 @@
+<template lang="pug">
+		v-app-bar(app, color='green', dark, dense)
+			v-toolbar-title JavaScript Code Smells Detector
+			v-spacer
+			v-btn(to='/knowledge-base', text) Knowledge base
+			v-btn(to='/upload', text) Code upload
+			v-btn(to='/editor', text) Code editor
+			v-btn(to='/', text) Home page
+</template>
+
+<script>
+export default {
+	name: 'BaseAppBar',
+};
+</script>
```

Comment:
You are correct about `Base` vs `The` prefix. Thanks for pointing that out, as coming from React and Svelte, I'm not aware of Vue-specific naming conventions. I disagree with the first statement. Even for a component with no logic it might make sense to give it a separate file for the sake of readability, as it makes you project tree more predictable. On top of that, AppBar won't remain a dumb component for long, as we are going to implement social login via GitHub OAuth and it will likely contain an appropriate button for login/logout.